packages/server/graphql/types/Organization.ts

```diff
@@ -85,6 +85,18 @@ const Organization: GraphQLObjectType<any, GQLContext> = new GraphQLObjectType<a
       description: 'The datetime the current billing cycle starts',
       resolve: resolveForBillingLeaders('periodStart')
     },
+    tierLimitExceededAt: {
+      type: GraphQLISO8601Type,
+      description: 'Flag the organization as exceeding the tariff limits by setting a datetime'
+    },
+    scheduledLockAt: {
+      type: GraphQLISO8601Type,
+      description: 'Schedule the organization to be locked at'
+    },
+    lockedAt: {
```

Comment:
+1 Might be good to add this to the DB type already as well
https://github.com/ParabolInc/parabol/blob/9e4726a29de46fad8869a155a9ee8228c1e405fc/packages/server/database/types/Organization.ts#L16